### ISSUE TITLE
Feature Request: Sort redirects by created and updated time/dates

### ISSUE THREAD

**Issue by SJNBham** [johngodley/redirection#2130]
Optional columns to sort redirects by creation and last updated time/dates. It will give a way to quickly review any new or recently changed redirects. It will also help identify really old redirects.

Tools > Redirection > Redirects > Custom Display > Created check box
Tools > Redirection > Redirects > Custom Display > Updated check box

Result would be optional columns in the redirects view that can be sorted chronologically by time/date.

![image](https://user-images.githubusercontent.com/11393597/72180106-4a6fa680-339b-11ea-873c-0febbb6f7970.png)

**Comment by johngodley** [comment]
I'll note that the time a redirect is created or updated is not currently stored, making this a difficult feature request.

However, there is a 'last redirected' time which may be a more useful metric in determining old redirects

**Comment by johngodley** [comment]
See #573 for storing the time

**Comment by SJNBham** [comment]
Yes - I noticed the creation and last modified date/time aren't stored in the database yet. This enhancement request would require that. Is there a concern about adding them to the database to enable this user interface feature?

**Comment by scrobbleme** [comment]
+1 We need this too.

We are creating redirections automatically and currently have ~72,000 redirections (don't ask, not my descission)...

But the intention is to just automatically delete the redirects after a perion of time (i.e. 1 year), no matter when the last access was.

_Addition_: Thanks to bots and crawlers, redirects are regularly triggered and the last access date is updated...

**Comment by stefanfisk** [comment]
I also need this and for the same reason as everyone else. My custom needs to delete old and unused redirects to lessen the mental burden of administering redirects. Right now we have to guess at how old an redirect is based on its ID and to call it flaky is an understatement...
